package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "ecommerce-app-predators-bn",
+  "version": "1.0.0",
+  "description": "predator ecommerce project",
+  "type": "module",
+  "main": "app.js",
+  "directories": {
+    "test": "tests"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node --experimental-modules server.js",
+    "dev": "nodemon --experimental-modules --es-module-specifier-resolution=node server.js",
+    "lint": "eslint ."
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/atlp-rwanda/ecommerce-app-predators-bn.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/atlp-rwanda/ecommerce-app-predators-bn/issues"
+  },
+  "homepage": "https://github.com/atlp-rwanda/ecommerce-app-predators-bn#readme",
+  "dependencies": {
+    "body-parser": "^1.20.2",
+    "cors": "^2.8.5",
+    "express": "^4.18.2",
+    "morgan": "^1.10.0",
+    "swagger-jsdoc": "^6.2.8",
+    "swagger-ui-express": "^4.6.2"
+  },
+  "devDependencies": {
```

Comment:
add this dependencies     "@babel/cli": "^7.21.0",
    "@babel/core": "^7.21.3",
    "@babel/preset-env": "^7.20.2",
    "jest": "^29.5.0"

Reply:
can you give me the names of the packages?

Reply:
npm install --save-dev @babel/cli
npm install --save-dev @babel/core
npm install @babel/preset-env --save-dev
npm install jest --global for jest,

Reply:
thank you @nili-alu